google-cloud-bigqueryconnection/src/main/java/com/google/cloud/bigquery/connection/v1beta1/ConnectionServiceClient.java

```diff
@@ -44,25 +36,14 @@
 import java.util.concurrent.TimeUnit;
 import javax.annotation.Generated;
 
-// AUTO-GENERATED DOCUMENTATION AND SERVICE
+// AUTO-GENERATED DOCUMENTATION AND CLASS.
 /**
  * Service Description: Manages external data source connections and credentials.
  *
  * <p>This class provides the ability to make remote calls to the backing service through method
  * calls that map to API methods. Sample code to get started:
  *
- * <pre>
- * <code>
```

Comment:
@miraleung Please keep us posted on when this will be added back.

Reply:
Will do, added you to [this tracking issue](https://github.com/googleapis/gapic-generator-java/issues/588)